client/components/Message/StatusMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, memo } from 'react';
+
+import { useUserData } from '../../hooks/useUserData';
+
+const StatusMessage: FC<{ uid: string }> = ({ uid }) => {
+	const data = useUserData(uid);
+
+	if (!data || !data.statusText) {
+		return null;
+	}
+
+	return (
+		<button className='message-custom-status' title={data.statusText}>
```

Comment:
I don't think this should be a button if it does nothing when clicked. I'd also prefer the use of the Icon component istead of an ASCII character.

Another thing is, I believe we should deprecate the use of old classnames. Perhaps style a box component to do this work?